modules/qna/src/backend/api.ts

```diff
@@ -14,7 +14,7 @@ export default async (bp: typeof sdk, botScopedStorage: Map<string, QnaStorage>)
   const csvUploadStatuses = {}
   const router = bp.http.createRouterForBot('qna')
 
-  router.get('/list', async (req, res) => {
+  router.get('/questions/list', async (req, res) => {
```

Comment:
why add the /list here ? 

get to `/questions` is explicit enough don't you think ?